python/pyarrow/parquet/__init__.py

```diff
@@ -3049,6 +3049,7 @@ def file_visitor(written_file):
         # extract non-file format options
         schema = kwargs.pop("schema", None)
         use_threads = kwargs.pop("use_threads", True)
+        chunk_size = kwargs.pop("chunk_size", None)
```

Comment:
Can you also catch `row_group_size`, as that is the documented keyword in `write_table` (it keeps `chunk_size` as an alias through `row_group_size = kwargs.pop('chunk_size', row_group_size)` inside `write_table`.
Probably giving precedence to `row_group_size` over `chunksize`

Reply:
Yes, will do!